tests/sass/class-name-format.scss

```diff
@@ -0,0 +1,96 @@
+.hyphenated-lowercase {
+  content: '';
+}
+
+.snake_case {
+  content: '';
+}
+
+.camelCase {
+  content: '';
+}
+
+.PascalCase {
+  content: '';
+
+  .Camel_Snake_Case {
+    content: '';
+
+    .SCREAMING_SNAKE_CASE {
+      content: '';
+    }
+  }
+}
+
+._with-leading-underscore {
+  content: '';
+}
+
+._does_NOT-fitSTANDARD {
+  @extend .snake_case;
+}
+
+.block-name__elem-name {
+  content: '';
+}
+
+.block {
+  &__element {
+    &--modifier {
```

Comment:
Are there any examples where multiple BEM blocks are used? Just to make sure it catches all of them.

i.e.

```
.test
  &__block
    &--mod
  &__block
    &--mod
    &--mod-two
  &--low-mod
```